src/helm/benchmark/scenarios/lextreme_scenario.py

```diff
@@ -423,7 +423,7 @@ def generate_instance(example, split: str):
                 if "multi_eurlex" in config:
                     input_text = ast.literal_eval(input_text)
                     assert isinstance(input_text, dict)
-                    languages = list(input_text.keys())
+                    languages = list([lang for lang, content in input_text.items() if content is not None])
```

Comment:
Came across the failure when running the legal-related benchmark. @JoelNiklaus feel free to take a look

Reply:
Thanks very much for fixing this!